Makefile

```diff
@@ -66,7 +66,11 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" POD_NAMESPACE=default go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" POD_NAMESPACE=default go test ./controllers/... -coverprofile cover.out
+
+.PHONY: e2e-test
+e2e-test: manifests generate fmt vet ## Run e2e-tests.
+	POD_NAMESPACE=default go test ./test/e2e/...
```

Comment:
Is it going to be executed only by CI?
If not, we could make it configurable, not sure if use the **default** as namesapce is a good practice.

Reply:
No, right now e2e tests are not going to be executed by the CI as there is no CI for e2e tests AFAICS - but the way I implemented it they should be as flexible as possible, such that we can also create a `Kind`-based github action to trigger them.

The only precondition is that the `odh-model-controller` must be deployed/running before triggering the `make e2e-test`.